app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -46,15 +46,20 @@ public AppResult(AppPojo appPojo) {
 
     @Override
     public View display(final Context context, int position, View v) {
-        if (v == null)
-            v = inflateFromId(context, R.layout.item_app);
+        if (v == null) {
```

Comment:
It would work, but performance would be much worse: for every line, the system would need to inflate a new row. This is something that is strongly discouraged.

Have a look here for details: http://stackoverflow.com/questions/11945563/how-listviews-recycling-mechanism-works